src/main/java/lotto/domain/stragegy/LottoNumberGenerator.java

```diff
@@ -0,0 +1,30 @@
+package lotto.domain.stragegy;
```

Comment:
```suggestion
package lotto.domain.strategy;
```
오타가 있는 것 같아요!
그런데 해당 클래스가 strategy 와 연관이 있을까요?